validator/storage_bucket_world_readable.rego

```diff
@@ -14,15 +14,15 @@
 # limitations under the License.
 #
 
-package templates.gcp.GCPStorageWorldReadableConstraintV1
+package templates.gcp.GCPStorageBucketWorldReadableConstraintV1
 
 deny[{
 	"msg": message,
 	"details": metadata,
 }] {
 	constraint := input.constraint
 	asset := input.asset
-	asset.asset_type == "google.cloud.storage.Bucket"
+	asset.asset_type == "google.cloud.storage/Bucket"
```

Comment:
This doesn't look quite right. It should be `storage.googleapis.com/Bucket` instead

Reply:
Fixed.